### PR TITLE
[IUPP-834]: Extend tesla-control to allow using access token value directly

### PR DESCRIPTION
**Pull request by Kirill-Kiyko**

### CODE REVIEW

File: pkg/cli/config.go [hunk 1]
```diff
@@ -399,6 +405,10 @@ func (c *Config) token() (string, error) {
 	if c.oauthToken != "" {
 		return c.oauthToken, nil
 	}
+    if c.TokenValue != "" {
```
Comment (bon-lucian) [inline pkg/cli/config.go]: *nit*: indentation seems to be incorrect here

File: cmd/tesla-control/commands.go [hunk 2]
```diff
@@ -491,6 +491,80 @@ var commands = map[string]*Command{
 			return car.ScheduleCharging(ctx, false, 0*time.Hour)
 		},
 	},
+    "departure-schedule": &Command{
```
Comment (bon-lucian) [inline cmd/tesla-control/commands.go]: *nit*: indentation seems to be incorrect here

**Review by bon-lucian — APPROVED** [review]

File: pkg/cli/config.go (hunk 1 above)
Comment (Kirill-Kiyko) [reply pkg/cli/config.go]: Fixed

File: cmd/tesla-control/commands.go (hunk 2 above)
Comment (Kirill-Kiyko) [reply cmd/tesla-control/commands.go]: Fixed

**Pull request merged**